Engine/src/camera.cpp

```diff
@@ -4,6 +4,10 @@
 #include <math.h>
 #include <application_state.h>
 
+namespace {
+    const float FACTOR_ORTHOGONAL_PROJECTION = 0.001f;
```

Comment:
No indent for namespaces:
```suggestion
const float FACTOR_ORTHOGONAL_PROJECTION = 0.001f;
```

Also break line after `namespace`.